grapes.js

```diff
@@ -71,17 +73,22 @@ class Grapes extends EventEmitter {
   }
 
   addDefaultGrapes () {
-    const g1 = new Grape({
-      dht_port: 20002,
-      dht_bootstrap: ['127.0.0.1:20001'],
-      api_port: 40001
-    })
-    const g2 = new Grape({
-      dht_port: 20001,
-      dht_bootstrap: ['127.0.0.1:20002'],
-      api_port: 30001
-    })
-    return [g1, g2]
+    const { ports } = this.opts
+    const {
+      g1Ports = {
+        dht_port: 20002,
+        dht_bootstrap: ['127.0.0.1:20001'],
+        api_port: 40001
+      },
+      g2Ports = {
+        dht_port: 20001,
+        dht_bootstrap: ['127.0.0.1:20002'],
+        api_port: 30001
+      }
+    } = { ...ports }
```

Comment:
can you make this an array: 

```
createGrapes({ ports: [
  {dht_port ....}
] )
```

so we are not exposing internal variables like "g1Port"?

Reply:
sure